Storage/Storage/Model/FeedbackType.swift

```diff
@@ -9,6 +9,10 @@ public enum FeedbackType: String, Codable {
     ///
     case productsM5
```

Comment:
I think that productsM5 can be removed from here since they are enabled for everyone.